File: Hache.Server/DTO/AuthSessionDTO.cs
using Hache.Server.Entities;

namespace Hache.Server.DTO
{
    public class AuthSessionDTO
    {
        public  string Token { get; set; }
        public string NombreUsuario { get; set; } 
        public string CorreoElectronico { get; set; } 
        public string NombreCompleto { get; set; } 
    }
}

File: hache.client/src/app/auth/serviceLogin/AuthSessionDTO.ts
export interface AuthSessionDTO {
  token: string;
  iD_Usuario: number;
  nombreUsuario: string;
  CorreoElectronico: string;
  nombreCompleto: string;
  iD_Local: number;
  tipoUsuario: {
  iD_TipoUsuario: number;
  nombre: string;
  }
}
